.mk/k8s.mk

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-NSM_NAMESPACE?=nsm-system
+NSM_NAMESPACE?=default
```

Comment:
Curious why the namespace change here?

Reply:
Is it not created by the helm chart?

Reply:
In helm charts, we are using the built-in object `Release.Namespace`. And here I think we can still use `NSM_NAMESPACE?=nsm-system`. NOTE: you can set NSM_NAMESPACE=default and invoke any `make` target :)